wordpress-mobile/WordPress-Android
# Quick Start: Track Quick Start Card Shown

Parent #16542

This PR tracks the quick start card shown event for my site tabs.
Internal Ref: p1652980710237019-slack-C0290FLA0RM

To test:

Prerequisite: Enable tracking using `Me` -> `App Settings` -> `Privacy settings` -> `Collect information`.

1. Launch the app. 
2. Logout/login and select "Show me around" when the quick start prompt is shown.
3. Select an existing site.
4. Go to `My Site` tab.
5. If "Home" tab is selected by default, notice that `my_site_dashboard_card_shown` event is tracked for quick start card (this tracking is added to shown qs card is shown on home/dashboard tab along with other dashboard cards):
    `🔵 Tracked: my_site_dashboard_card_shown, Properties: {"type":"quick_start","default_tab_experiment":<value>,"subtype":"quick_start_existing_site"}`
6. If "Home" tab is not selected by default, select "Home" tab using "Initial Screen" setting.
7. Notice that `my_site_dashboard_card_shown` event is tracked for quick start card.
8. Notice that following events are also tracked along with `tab` and `site_type` properties (this tracking is added to track qs card shown event for site menu tab as well):
    ` 🔵 Tracked: quick_start_card_shown, Properties: {"tab":"site_menu","site_type":"existing_site","default_tab_experiment":<value>}`
    `🔵 Tracked: quick_start_card_shown, Properties: {"tab":"dashboard","site_type":"existing_site","default_tab_experiment":"<value>}`


## Regression Notes
1. Potential unintended areas of impact 
N/A

3. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

4. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
